### DISCUSSION TITLE
Windows Alt+N doesn't work in Send Later pop-up

### DISCUSSION THREAD

**Discussion by JimCarls** [Extended-Thunder/send-later#347] [Q&A] [answered]
I've searched for this issue and did not find a clear discussion, however in the new version (new as in not the legacy one), the "Send Now" button has never activated when I tried to use it after invoking Send Later with Ctrl-Enter, despite having the underlined "N" in the label. Thanks!

**Comment by jph-sendlater** [comment, marked as answer]
I'm not sure, because I'm not a Windows user, but you might need to use Alt+Shift+N

I've seen conflicting information, but most documentation says that Mozilla browsers (and I would expect Thunderbird by extension), associate Alt+Shift with the `accesskey` property on Windows. e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey

**Reply by JimCarls** [reply]
Thanks! The Alt+Shift combination worked.
